.build/tar/generate_tar.sh

```diff
@@ -16,7 +16,7 @@
 
 CONFIG_FILE=${CONFIG_FILE:-config-example.yaml}
 EXPORTERS_DIRECTORY=prometheus_exporters
```

Comment:
nit: this directory name should be singular to be consistent with the change below, or the config directory name should be plural